src/HelixServer.js

```diff
@@ -74,9 +74,15 @@ class HelixServer {
           .then(boundResolve)
           .then(executeTemplate)
           .then((result) => {
-            esi.process(result.response.body).then((body) => {
-              res.send(body);
-            });
+            if (result instanceof Error || (result.response && result.response.error)) {
+              const stack = result.stack || result.response.error.stack;
```

Comment:
shouldn't you also check if `result.response.error` is an `Error` ?(or at least has a stack)